src/gameHelper.tsx

```diff
@@ -0,0 +1,40 @@
+export const STAGE_WIDTH: number = 12;
+export const STAGE_HEIGHT: number = 20;
+
+
+
+interface point {
+    x: number
+    y: number
+}
+
+interface IPlayer {
+    pos: point
+    tetromino: (string | number)[][],
+    collided: boolean,
+}
+
+
+export const createStage = ():Array<Array<Array<string | number>>>=> {
+var stage:Array<Array<Array<string | number>>>= new Array(STAGE_HEIGHT);
+    for (var i = 0; i < stage.length; i++) {
+        stage[i] = new Array(STAGE_WIDTH).fill([0, 'clear'])
+    }
+    return stage
+
+}
+
+export const checkCollision = (player:IPlayer, stage:Array<Array<Array<string | number>>>,moveX:number,moveY:number ) => {
```

Comment:
Instead of mentioning `Array<Array<Array<string | number>>>` this everywhere can you create a type or interface & refer to it?